Runtimes/Resync.cmake

```diff
@@ -160,6 +160,7 @@ copy_files(public/Platform Overlay/Windows/clang
     ucrt.modulemap
     winsdk_um.modulemap
     winsdk_shared.modulemap
+    WinRT.modulemap
```

Comment:
`winsdk_winrt` seems like a more consistent name? it would be ideal to `import WinSDK.WinRT` as opposed to `import WinRT`. the latter can cause lots of collisions with other WINMD based projection tools which will generate code in the `WinRT` module

Reply:
there are other submodules in `WinSDK`, so that should be fine?

Reply:
>For this reason, I'm going to propose splitting WinRT headers into many top-level modules, e.g. having a top-level module WindowsXyz per windows.xyz.*.h. But that is out of scope of this PR ;)

Does the fact that namespaces in WinRT can have cyclical dependencies affect this at all?